src/librustc_passes/diagnostics.rs

```diff
@@ -129,16 +129,18 @@ Please note that negative impls are only allowed for auto traits.
 "##,
 
 E0265: r##"
+#### Note: this error code is no longer emitted by the compiler.
```

Comment:
I've been removing unused errors entirely before. Is there any reason not to do that?

Reply:
Not sure, but E0001 and E0002 are being kept...

Reply:
We're not consistent in doing this. I don't even like error codes *at all*.

Reply:
I think we *usually* leave a tombstone. It doesn't matter so much for these older errors, but it's particularly useful if the error is the last one that was made, so that it doesn't get reused.

Reply:
(I too would sort of like to reform error codes somewhat, though I'm not entirely sure to what.)